lib/sidekiq_publisher.rb

```diff
@@ -1,16 +1,15 @@
 # frozen_string_literal: true
 
-require "active_support"
+require "active_support/core_ext/numeric/time"
```

Comment:
The `require "active_record"` in `sidekiq_publisher/job` was indirectly doing this for us. Now that we aren't doing that, we have to explicitly do this ourselves (it's used on line 16 below).

While I was figuring this out, I figured it was worth more specifically targeting what is needed from Active Support throughout the gem. If the preference is to keep the wholesale `require "active_support"`, just let me know!

Reply:
`require "active_support"` is not the wholesale require. It is documented (https://guides.rubyonrails.org/active_support_core_extensions.html) that you should call `require "active_support"` and then specific extensions. 

The require for everything would be `require "active_support/all"`.

Reply:
@tjwp TIL! Just updated the first commit here to keep the `require "active_support"`, and then the specific extensions needed.